subsys/bluetooth/shell/bt.c

```diff
@@ -738,6 +739,38 @@ static int cmd_id_select(int argc, char *argv[])
 	return 0;
 }
 
+static int cmd_ll_get_addr(int argc, char *argv[])
+{
+	u8_t addr_type;
+	const char * str_type;
+	bt_addr_t addr;
+	char str_addr[BT_ADDR_STR_LEN];
+
+	if (argc < 2)
+		return -EINVAL;
+
+	str_type = argv[1];
+	if (!strcmp(str_type, "random"))
+	{
+		addr_type = 1;
+	}
+	else if (!strcmp(str_type, "public"))
+	{
+		addr_type = 0;
+	}
+	else
+	{
+		return -EINVAL;
+	}
```

Comment:
All of the above three branches need fixes to the use of {}. It should be:
```
if (...) {
        ...
} else if (...) {
        ...
} else {
        ...
}
```

Reply:
Done